proto/cosmos/tx/v1beta1/service.proto

```diff
@@ -42,6 +42,8 @@ message GetTxsEventRequest {
   repeated string events = 1;
   // pagination defines an pagination for the request.
   cosmos.base.query.v1beta1.PageRequest pagination = 2;
+  // order_by defines events sorting order
+  string order_by = 3;
```

Comment:
should we use an enum?

If not, it would be good if the docs show the 2 possible values.

Reply:
+1 for enum